internal/impl/ockam/output_kafka.go

```diff
@@ -0,0 +1,150 @@
+package ockam
+
+import (
+	"context"
+	"errors"
+	"strings"
+
+	"github.com/benthosdev/benthos/v4/internal/impl/kafka"
+	"github.com/benthosdev/benthos/v4/public/service"
+)
+
+// this function is, almost, an exact copy of the init() function in ../kafka/output_kafka_franz.go
+func init() {
+	err := service.RegisterBatchOutput("ockam_kafka", ockamKafkaOutputConfig(),
+		func(conf *service.ParsedConfig, mgr *service.Resources) (
+			output service.BatchOutput,
+			batchPolicy service.BatchPolicy,
+			maxInFlight int,
+			err error,
+		) {
+			if maxInFlight, err = conf.FieldInt("max_in_flight"); err != nil {
+				return
+			}
+			if batchPolicy, err = conf.FieldBatchPolicy("batching"); err != nil {
+				return
+			}
+			output, err = newOckamKafkaOutput(conf, mgr.Logger())
+			return
+		})
+	if err != nil {
+		panic(err)
+	}
+}
+
+func ockamKafkaOutputConfig() *service.ConfigSpec {
+	return kafka.FranzKafkaOutputConfig().
+		Summary("Ockam").
+		Field(service.NewStringField("ockam_enrollment_ticket").Optional()).
+		Field(service.NewStringField("ockam_identity_name").Optional()).
+		Field(service.NewStringField("ockam_allow_consumer").Default("self")).
+		Field(service.NewStringField("ockam_route_to_consumer").Default("/ip4/127.0.0.1/tcp/6262"))
+}
```

Comment:
I'll push some documentation for these tomorrow with some examples on how to use them.